packages/react-router-dom/modules/__tests__/Link-test.js

```diff
@@ -89,3 +91,18 @@ describe('A <Link> underneath a <HashRouter>', () => {
     })
   })
 })
+
+describe('resolving a relative Link', () => {
+  it('can use resolveLocation to resolve relative to parent Route', () => {
+    const node = document.createElement('div')
+    ReactDOM.render((
+      <MemoryRouter initialEntries={[ '/state' ]}>
+        <Route path='/state' render={({ match }) => (
+          <Link to={resolveLocation('CO', match.url)}>Colorado</Link>
+        )} />
+      </MemoryRouter>
+    ), node)
+    const a = node.getElementsByTagName('a')[0]
+    expect(a.getAttribute('href')).toEqual('/state/CO')
+  })
+})
```

Comment:
It isn't _really_ necessary, but serves as a filler until the `<Link>` natively resolves relative locations. I mostly wrote it as a sanity check.